extensions/amp-brid-player/validator-amp-brid-player.protoascii

```diff
@@ -49,7 +49,10 @@ tags: {  # <amp-brid-player>
   attrs: {
     name: "data-playlist"
     mandatory_oneof: "['data-outstream', 'data-playlist', 'data-video']"
-    value_regex: "[0-9]+"
```

Comment:
I see in the example that `data-playlist` has text which may be why this restriction is being removed. However removing this restriction allows the value to be anything (including nothing). That doesn't seem ideal. Would `(\\d|\\w)+` be sufficient for a valid value? Are there other valid characters in a non-id playlist name?

May also need to update the documentation for `data-playlist` stating what is allowed. Currently it states `ID` which till now was just a number.

Reply:
Would it be OK if we put `.+` because we would like to enable users to put anything there? Will update the documentation as you suggested. Thanks.

Reply:
That is fine. I just want to be sure it's what one would want as it's to prevent someone from doing something wrong unintentionally.

Reply:
The requested changes are implemented now.